README.md

```
@@ -1,14 +1,43 @@
+<!-- ALL Links -->
+[contributors-shield]: https://img.shields.io/github/contributors/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=4C8BF5
+[contributors-url]: https://github.com/Learnathon-By-Geeky-Solutions/binary-brains/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=4C8BF5
+[forks-url]: https://github.com/Learnathon-By-Geeky-Solutions/binary-brains/network/members
+[stars-shield]: https://img.shields.io/github/stars/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=4C8BF5
+[stars-url]: https://github.com/Learnathon-By-Geeky-Solutions/binary-brains/stargazers
+[issues-shield]: https://img.shields.io/github/issues/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=FF6F61
+[issues-url]: https://github.com/Learnathon-By-Geeky-Solutions/binary-brains/issues
+[license-shield]: https://img.shields.io/github/license/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=28A745
+[license-url]: https://github.com/Learnathon-By-Geeky-Solutions/binary-brains/blob/master/LICENSE
+[sonarqube-shield]: https://img.shields.io/static/v1?label=Quality%20Gate&message=Passed&color=28A745&style=for-the-badge
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "sonarqube-shield"

(MD053, link-image-reference-definitions)